functions/zram.bash

```diff
@@ -74,6 +74,12 @@ init_zram_mounts() {
     if ! cond_redirect install -m 644 "$zramInstallLocation"/openhabian-zram/ro-root.sh /usr/local/share/zram-config/ro-root.sh; then echo "FAILED (ro-root)"; return 1; fi
     if cond_redirect install -m 644 "$zramInstallLocation"/openhabian-zram/zram-config.logrotate /etc/logrotate.d/zram-config; then echo "OK"; else echo "FAILED (logrotate)"; return 1; fi
 
+    if [[ -f /etc/systemd/system/find3server.service ]]; then
+      echo -n "$(timestamp) [openHABian] Adding FIND3 to ZRAM... "
+      if ! cond_redirect systemctl stop zram-config.service; then echo "FAILED (stop service)"; return 1; fi
```

Comment:
no stop and no start but a one-shot call of /usr/local/bin/zram-config start

Reply:
Done